server/src/main/java/org/elasticsearch/action/admin/indices/template/put/PutIndexTemplateResponse.java

```diff
@@ -47,4 +49,14 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         writeAcknowledged(out);
     }
+
+    private static final ConstructingObjectParser<PutIndexTemplateResponse, Void> PARSER;
+    static {
+        PARSER = new ConstructingObjectParser<>("put_index_template", true, args -> new PutIndexTemplateResponse((boolean) args[0]));
+        declareAcknowledgedField(PARSER);
+    }
+
+    public static PutIndexTemplateResponse fromXContent(XContentParser parser) {
+        return PARSER.apply(parser, null);
```

Comment:
Although this response is super simple, we may want to add a PutIndexTemplateResponseTests similar to UpdateSettingsResponseTests ?